apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/Actor/Models/CreateProcessDelegationsInput.cs

```diff
@@ -14,11 +14,11 @@
 
 using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
 
-namespace Energinet.DataHub.WebApi.GraphQL.Types.Process;
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.Actor.Models;
 
 public sealed record CreateProcessDelegationsInput(
         Guid DelegatedFrom,
         Guid DelegatedTo,
-        ICollection<string> GridAreas,
-        ICollection<DelegatedProcess> DelegatedProcesses,
+        IEnumerable<Guid> GridAreasIds,
```

Comment:
nitpick: should it be `GridAreaIds` (no "s" after GridArea)?

Reply:
Nope, removed now :-)